# Publish all joint states -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50  

# Position Controllers ---------------------------------------
#PID values might need more tuning
RHip_yaw_position_controller:
  type: effort_controllers/JointPositionController
  joint: RHip_yaw
  pid: {p: 30.0, i: 0.1, d: 18.0}
RHip_roll_position_controller:
  type: effort_controllers/JointPositionController
  joint: RHip_roll
  pid: {p: 10.0, i: 1.0, d: 27.0}
RHip_pitch_position_controller:
  type: effort_controllers/JointPositionController
  joint: RHip_pitch
  pid: {p: 20.0, i: 0.1, d: 18.0}
RKnee_pitch_position_controller:
  type: effort_controllers/JointPositionController
  joint: RKnee_pitch
  pid: {p: 20.0, i: 0.05, d: 10.0}
RAnkle_pitch_position_controller:
  type: effort_controllers/JointPositionController
  joint: RAnkle_pitch
  pid: {p: 10.0, i: 1.25, d: 28.0}
RAnkle_roll_position_controller:
  type: effort_controllers/JointPositionController
  joint: RAnkle_roll
  pid: {p: 10.0, i: 1.0, d: 27.0}




LHip_yaw_position_controller:
  type: effort_controllers/JointPositionController
  joint: LHip_yaw
  pid: {p: 30.0, i: 0.1, d: 18.0}
LHip_roll_position_controller:
  type: effort_controllers/JointPositionController
  joint: LHip_roll
  pid: {p: 10.0, i: 1.0, d: 27.0}
LHip_pitch_position_controller:
  type: effort_controllers/JointPositionController
  joint: LHip_pitch
  pid: {p: 20.0, i: 0.1, d: 18.0}
LKnee_pitch_position_controller:
  type: effort_controllers/JointPositionController
  joint: LKnee_pitch
  pid: {p: 25.0, i: 0.05, d: 20.0}
LAnkle_pitch_position_controller:
  type: effort_controllers/JointPositionController
  joint: LAnkle_pitch
  pid: {p: 10.0, i: 1.25, d: 28.0}
LAnkle_roll_position_controller:
  type: effort_controllers/JointPositionController
  joint: LAnkle_roll
  pid: {p: 10.0, i: 1.0, d: 27.0}
